### DISCUSSION TITLE
other languages support

### DISCUSSION THREAD

**Discussion by sebastianwebber** [goreleaser/goreleaser#1689] [Q&A] [answered]
Hi there,

There is any plan to support other languages or there is any way to replace the build steps for custom commands?

I'm would love to use go-release to package other non-Go applications.

if this is something that the project has an interest, please let me know. I will love to contribute.

Thanks!

**Comment by caarlos0** [comment, marked as answer]
we have some stuff lay out for this, the idea was to use go plugins, but they don't work on some platforms and also there was little interest on it... so we just droped it for the time being...

**Reply by sebastianwebber** [reply]
my general idea is to add a new "tag" to de builds config, like this:

```yaml
----
...
builds:
  - shell:
     -  command 1
     -  command 2
     -  command 3
  - shell:
     -  command 1
     -  command 2
     -  command 3
...
```

For the case that the `shell` is set, the default `go build` will not run. 

What is the best way to propose something like this? with a PR?

**Reply by caarlos0** [reply]
> What is the best way to propose something like this? with a PR?

yes, but I don't think we will accept this specifically, as it would increase the scope of goreleaser a lot (and its already very big)...

**Reply by sebastianwebber** [reply]
sure, the entire idea is to discuss and check if that is of the interest of the project.

Thanks for your answers. =)
